user-journey-tests/page-objects/newAddressPage.js

```diff
@@ -168,6 +168,19 @@ class NewAddressPage extends Page {
     }
   }
 
+  async clearFormFields(fields = {}) {
+    for (const [field, value] of Object.entries(fields)) {
+      const fieldMappings = this.getFieldMappings()
```

Comment:
Just a tiny thing, but move it one line up & it'll just run once.

Reply:
Great spot